src/main/steps/common/form/form.njk

```diff
@@ -107,7 +107,7 @@
             name: "saveAndComeLater",
             value: true,
             attributes: {
-              id: 'main-form-save-as-draft'
+              id: 'main-form-save-and-come-back-later'
```

Comment:
Why is the id changed? Is the id used in code any where? Does it impact any of the functionality? Have you tested the changes?

Reply:
Its duplicated in same file so changed it 
Its not used anywhere across the application